doc/devel/contributing.rst

```diff
@@ -437,6 +437,70 @@ forced to use ``**kwargs``.  An example is
       elif len(args) == 1:
          ...etc...
 
+.. _using_logging:
+
+Using logging for debug messages
+--------------------------------
+
+Matplotlib uses the standard python `logging` library to write verbose
```

Comment:
@anntzer The doc build gives 

```
Warning, treated as error:
/home/circleci/project/doc/faq/troubleshooting_faq.rst:117:Unknown target name: "logging".
Building HTML failed.
Exited with code 1
```

Did I do this wrong or is our map not working?